docs/jaxpr.rst

```diff
@@ -293,44 +262,25 @@ contains a constant ``jnp.ones(1)`` that is hoisted as a `constvar`
 >>> def func8(arg1, arg2):  # arg2 is a pair
 ...   return lax.cond(arg1 >= 0.,
 ...                   lambda xtrue: xtrue[0],
-...                   lambda xfalse: jnp.ones(1) + xfalse[1],
+...                   lambda xfalse: jnp.array([1]) + xfalse[1],
 ...                   arg2)
 ...
 >>> print(make_jaxpr(func8)(5., (jnp.zeros(1), 2.)))
-{ lambda f ; a b c.
-  let d = ge a 0.0
-      e = convert_element_type[ new_dtype=int32
-                                old_dtype=bool ] d
-      g = cond[ branches=( { lambda  ; c a b.
-                             let d = add c b
-                             in (d,) }
-                           { lambda  ; e_ a b.
-                             let 
+{ lambda a ; b c d.
+  let e = ge b 0.0
+      f = convert_element_type[ new_dtype=int32
+                                old_dtype=bool ] e
+      g = cond[ branches=( { lambda  ; a b c.
+                             let d = convert_element_type[ new_dtype=float32
+                                                           old_dtype=int32 ] a
+                                 e = add d c
+                             in (e,) }
+                           { lambda  ; f_ a b.
+                             let
                              in (a,) } )
-                linear=(False, False, False) ] e f b c
+                linear=(False, False, False) ] f a c d
   in (g,) }
 
-The top-level jaxpr has one `constvar` ``f`` (corresponding to
```

Comment:
Do you plan to remove all the text here?

Reply:
I created #4293 so I will follow up on improving these docs.